### DISCUSSION TITLE
Dolphin MMJR2 Radiant dawn configuration/settings.

### DISCUSSION THREAD

**Discussion by Silverhandknight** [Medard22/Dolphin-MMJR2-VBI#871] [General]
Hello, First of all English is not my first language, So if you don't understand what I said please feel free to ask me.

I don't know if i can post this here or not, if I make mistake please let me know, I will delete it.

If you have any trouble with fire emblem configuration. Because it's very slow and you got low fps, I will share my own configuration. 

My device is Xiaomi redmi note 7 4/64, Snapdragon 660.
It works fine on my device, When i play i got around 30-45 fps on battle and 60 fps on menu. Depend on your device maybe you got more, I use 1x resolution because my device cant handle 2x resolution, if your device can handle, it will fix blurry text in game, i use 1x resolution, but i rarely get blurry text.

My configuration works for many gamecube & Wii games, But depends on what game, you need to change the settings a bit.
 
I use the latest mmjr2 right now version U23.12.18-20789.
Please let me know if this configuration works with your device.

Im sorry for the low quality video, I had to compress it so I could upload here.

https://github.com/user-attachments/assets/80138644-305f-4aea-a7eb-0e825c6ca88a

**Comment by Medard22** [comment]
Feel free to leave it here. Every device is different, even devices with same SoC may have different Android versions and GPU drivers. Some settings can speed up the emulation on some devices, while slowing them on others. The best way is to establish a base default configuration that works for most of the games on your device, then make a custom config for games that has issues and test what works and what doesn't.

**Reply by Medard22** [reply]
Also Snapdragon 660 is a mid 2017 chip. Like T618 in RP3+ it will struggle with Gamecube and Wii emulation. Some games may work, but a lot of them will not be playable.

**Reply by Silverhandknight** [reply]
Thanks for replying me. 
Yeah, I agree with you, I just wanted to share this because I cant seem to find the correct configuration for my mid end phone, and maybe another people have the same issue . 
I have another phone with snapdragon 8 gen 2, I use the default settings, and its working without any problem in many game. MMJR2 Is very good even my mid end phone can run it. If I use the official dolphin on my old phone it's not running well.

**Reply by Medard22** [reply]
Yeah, that's one of the reasons I'm trying to keep it alive. It works great on my Retroid and as long as it's useful to others I'm happy maintaining it.

**Reply by Silverhandknight** [reply]
Keep up the good work, Man! 👍
